media/css/mozorg/home/takeover.less

```diff
@@ -0,0 +1,220 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+@mozRed: #c33d32;
+@backgroundColor: #f89c24;
+@foregroundColor: #fff;
+@optionInactive: #90939d;
+@optionActive: #259BEB;
+
+.fundraising-takeover * {
+    .border-box;
+}
+
+.fundraising-takeover {
+    position: absolute;
+    top: 0;
+    left: 0;
+    background: @backgroundColor url('/media/img/home/takeover/hart.svg') center top no-repeat;
+    .background-size(100%);
+    color: @foregroundColor;
+    width: 100%;
+    height: 720px;
```

Comment:
Is there a reason why this needs a fixed height? Given that string lengths can push content down, does it make life easier to remove this hard-coded value and allow `auto`?